.github/workflows/ci-ubuntu-combined.yml

```diff
@@ -179,11 +179,11 @@ jobs:
       - name: Artifact Idris2
         uses: actions/upload-artifact@v2
         with:
-          name: idris2-nightly
+          name: idris2-nightly-chez
           path: ~/.idris2/
 
   build-api:
-    needs: build-previous-version
+    needs: self-host-previous-version
```

Comment:
Wait. Do we need the self-host version or would the build-bootstrap-chez be enough?